scripts/log.ps1

```diff
@@ -143,7 +143,9 @@ function Log-Cancel {
     if ($IsWindows) {
         try {
             wpr.exe -cancel -instancename $InstanceName 2>&1
+            $global:LASTEXITCODE = 0
```

Comment:
Why in both places, instead of just after the try/catch?

Reply:
Fixed.